src/main/webapp/feedback.jsp

```diff
@@ -0,0 +1,74 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%
+  boolean canOpen = (boolean) request.getAttribute("feedbackOpen");
+  String role = (String) request.getAttribute("role");
+  pageContext.setAttribute("feedbackOpen", canOpen);
+  pageContext.setAttribute("role", role); 
+%>
+
+
+<c:choose>
+  <c:when test= "${canOpen == false}">
+    <h2 style="text-align: center">You may not submit feedback yet.</h2>
+  </c:when>
+  <c:otherwise>
+    <c:choose>
+      <c:when test= "${role == 'Interviewee'}">
+        <h1 class="text-center">Please submit your feedback for your interviewer below</h1>
```

Comment:
Titles should not be this long. Please just change it to "Feedback" and put the rest of the information in a subtitle

Reply:
Done